content/engineering/ruby.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Ruby Guide
+sidenav: true
+permalink: /engineering/ruby/
+tags: engineering
+layout: layouts/page
+eleventyNavigation: 
+  parent: engineering_languages
+  key: Ruby
+  order: 5
+  title: Ruby
+---
+Help us make this section better by
+[submitting an issue](https://github.com/18F/development-guide) or joining us
+in the [#ruby](https://18f.slack.com/messages/ruby/) channel!
+
+A guide for writing and maintaining Ruby and Rails applications
+
+## Style Guide
+
+Follow the [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide) and
+enforce it via static analysis tools such as [Code Climate] and [Rubocop]. You
+can copy the [recommended Rubocop configuration](rubocop.yml) in your Ruby
```

Comment:
Another asset that would need to get covered in https://github.com/18F/TLC-crew/issues/337

Reply:
asset ticket :)